assets/javascripts/discourse/initializers/extend-for-translate-button.js

```diff
@@ -123,24 +134,6 @@ function customizeWidgetPostMenu(api) {
       post.set("translated_text", "");
     }
   });
-
-  api.addPostMenuButton("translate", (attrs, state) => {
-    if (!attrs.can_translate) {
-      return;
-    }
-
-    const [action, title] = !state.isTranslated
-      ? ["translate", "translator.view_translation"]
-      : ["hideTranslation", "translator.hide_translation"];
-
-    return {
-      action,
-      title,
-      icon: "globe",
-      position: "first",
-      className: state.isTranslated ? "translated" : null,
-    };
-  });
```

Comment:
It looks perfect 🎊 
My only confusion is, why we are removing this button? I think we still need it to translate posts "the old way"